src/js/containers/LimitOrder/LimitOrderTable.js

```diff
@@ -623,17 +694,31 @@ export default class LimitOrderTable extends Component {
       )
     } else if (title === "status") {
       return (
-        <Dropdown active={this.state.statusFilterVisible} onHide={e => this.togglingStatusFilter()}>
-          <div>
-            <span>{(this.props.translate("limit_order.status") || "Status").toUpperCase()}</span>
-            <div className="drop-down">
-              <img src={require("../../../assets/img/v3/price_drop_down.svg")}/>
+          <Dropdown active={this.state.statusFilterVisible} onHide={e => this.togglingStatusFilter()}>
```

Comment:
When we use () => func() and when we use e => func(). I see many places are different

Reply:
With only 1 param, the parentheses can be removed